lib/perl/Genome/Model/Tools/Picard/MarkDuplicates-filehandles.t

```diff
@@ -15,17 +15,14 @@ my $cmd = $class->create(
     output_file => '/tmp/bar',
     metrics_file => '/tmp/baz',
     temp_directory => '/tmp',
-    use_version => "1.34"
+    use_version => "1.36"
     );
 
-my $fh = $cmd->get_max_filehandles_param;
-like($fh, qr/^MAX_FILE_HANDLES=[0-9]+$/, "Version 1.34 sets MAX_FILE_HANDLES");
+my $fh = $cmd->build_cmdline_string;
+like($fh, qr/MAX_FILE_HANDLES=[0-9]+$/, "Version > 1.34 sets MAX_FILE_HANDLES");
```

Comment:
Should this be a `>=`?  Reading the new version of the test I wasn't sure what `1.34` should do.